testscommon/factory/transactionCoordinatorFactoryStub.go

```diff
@@ -3,19 +3,16 @@ package factory
 import (
 	"github.com/multiversx/mx-chain-go/process"
 	"github.com/multiversx/mx-chain-go/process/coordinator"
+	"github.com/multiversx/mx-chain-go/testscommon"
 )
 
 // TransactionCoordinatorFactoryStub -
 type TransactionCoordinatorFactoryStub struct {
```

Comment:
Duplicated with the one from `testscommon/factory/transactionCoordinatorFactoryMock.go`

Reply:
removed